voyager_archive/templates/voyager_archive/marc_display.html

```diff
@@ -11,18 +11,33 @@
 </form>
 <hr/>
 
-{% comment %}
-Raw Marc dump
-{% endcomment %}
+<div class="row">
+    <div class="column">
+        <h2>Record Leader: {{ marc_record.leader }}</h1>
+        <table class="data-display">
+            <tr>
+                <th>Tag</th>
+                <th>Indicators</th>
+                <th>Data and Subfields</th>
+            </tr>
+            {% for field in marc_record.fields %}
+            <tr>
+                <td>{{ field.tag }}</td>
+                <td><span class='indicators'>{{ field.indicators }}</span></td>
```

Comment:
Blank indicators are not clear, at least when the first indicator is blank.  For example, in bib 9411434, the 050 field has 1st ind blank, 2nd ind 4.  The template currently displays that without the blank, so the 2nd ind 4 is aligned with the 1st ind of the 245 a couple of fields later.  See screenshots below.

You mentioned using a fixed width font, but I'm not seeing that defined in the CSS.  That could be one solution.

Other options:
* Change blank indicators to underscore for display
* Change blank indicators to `&nbsp;` (non-breaking space entity)
* Use separate table columns for each indicator after all

![indicators_bad](https://user-images.githubusercontent.com/2213836/194187323-2aa73555-724f-4b44-abee-66a39de0b821.png)

![indicators_good](https://user-images.githubusercontent.com/2213836/194187334-c632f138-7574-4e7e-99a1-5dcb094aede8.png)